src/models/PodData/PodData.ts

```diff
@@ -0,0 +1,37 @@
+import { PacketUpdate } from "@adapters/PacketUpdate";
+import { Board } from "@models/PodData/Board";
+import { Packet } from "@models/PodData/Packet";
+
+export class PodData {
+  boards: Board[];
+  lastBatchIDs: number[] = [];
+
+  constructor(boards: Board[]) {
+    this.boards = boards;
+  }
+
+  public setLastBatchIDs(updates: PacketUpdate[]) {
+    this.lastBatchIDs = updates.map((update) => update.id);
+  }
+
+  public update(updates: PacketUpdate[]) {
```

Comment:
shouldn't this receive a map? the back sends a map as the packet update where the key is the packet id and the values are the packets

Reply:
We should check how go serializes maps

Reply:
as JSON objects